netty/src/main/java/io/grpc/netty/AbstractNettyHandler.java

```diff
@@ -50,6 +54,7 @@
   private static long GRACEFUL_SHUTDOWN_TIMEOUT = SECONDS.toMillis(5);
   private int initialConnectionWindow;
   private ChannelHandlerContext ctx;
+  private boolean flowControlOn = false;
```

Comment:
Flow control is always on. Let's figure out a different name. autoAdjustFlowControl, autoTuneFlowControl, autoIncreaseFlowControl, pingManagedFlowControl? Or something else.